docs/kit/logistics/kit-return.md

```diff
@@ -0,0 +1,19 @@
+# Kit Return
+
+## At the competition
+
+At the end of the competition, teams are asked to return their kits to reception. Appendix A.1 of the [rules](https://studentrobotics.org/docs/rules/) notes the list of items expected back, items on this list are not expected back, but it's fine for them to be. At reception, the contents of the RUB should be checked to ensure it contains at least all the items on this list. Ensure there is nothing else left in the RUB the team may want back.
```

Comment:
This block has two modes of instruction:
- for someone planning the return (mention of the rules etc,)
- for someone doing the task (ensure RUB is empty)

I suggest breaking these into separate parts.